Program/View/interface_renderer.py

```diff
@@ -1,6 +1,7 @@
 import tkinter as tk
 from tkinter.scrolledtext import ScrolledText
 from Validators.validator_runner import ValidatorRunner
+from FileHandling.file_logic import FileLogic
```

Comment:
Maybe switch file handling and validators so they are alphabetical, but this is a NIT